### PR TITLE
Self-Hosted Custom CA Update

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the instructions for installing a custom CA when deploying Palette or VerteX in an airgap environment.

- [VerteX](https://deploy-preview-3061--docs-spectrocloud.netlify.app/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions/)
- [Palette](https://deploy-preview-3061--docs-spectrocloud.netlify.app/enterprise-version/install-palette/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions/)

## Changed Pages

💻 [Preview URL for Page]()

## Jira Tickets

🎫 [DOC-1224](https://spectrocloud.atlassian.net/browse/DOC-1224)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1224]: https://spectrocloud.atlassian.net/browse/DOC-1224?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md [hunk 1]
```diff
@@ -238,23 +238,42 @@ If you are working in Vim, press `i` to enter insert mode in the text editor. Pr
     ```
 
 19. If you have custom SSL certificates you want to apply to the image and pack registry, and the Spectro Cloud
-    Repository, copy the custom SSL certificates, in base64 PEM format, to the airgap support VM. The custom
-    certificates must be placed in the **/opt/spectro/ssl** folder. Include the following files:
-
-    - **server.crt**
-    - **server.key**
+    Repository, copy the custom SSL certificates, in base64 PEM format, to the airgap support VM.
```
Comment (caroldelwing) [inline docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md:241]:
```suggestion
    Repository, copy the custom SSL certificates in base64 PEM format to the airgap support VM.
```
maybe? :)

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md [hunk 2]
```diff
@@ -238,23 +238,42 @@ If you are working in Vim, press `i` to enter insert mode in the text editor. Pr
...
     :::warning
 
-    The custom SSL certificates must be in base64 PEM format. If you have custom SSL certificates in a different format,
-    convert them to base64 PEM format before copying them to the support VM. The airgap setup process also expects the
-    files to be named **server.crt** and **server.key**.
+    Three items you need to keep in mind when providing custom SSL certificates:
+
+    - The custom SSL certificates must be in base64 PEM format. If you have custom SSL certificates in a different
+      format, convert them to base64 PEM format before copying them to the support VM. The airgap setup process also
+      expects the files to be named **server.crt** and **server.key**.
```
Comment (caroldelwing) [inline docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md:252]: Same question hehe Which files?

**Review by caroldelwing — APPROVED** [review]
Approved with a tiny question :)
I could not leave a comment there, but there is a box in step 17 of the VerteX page that is not aligned.

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md [hunk 3]
```diff
@@ -237,23 +237,42 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
...
     :::warning
 
-    The custom SSL certificates must be in base64 PEM format. If you have custom SSL certificates in a different format,
-    convert them to base64 PEM format before copying them to the support VM. The airgap setup process also expects the
-    files to be named **server.crt** and **server.key**.
+    Three items you need to keep in mind when providing custom SSL certificates:
+
+    - The custom SSL certificates must be in base64 PEM format. If you have custom SSL certificates in a different
+      format, convert them to base64 PEM format before copying them to the support VM. The airgap setup process also
+      expects the files to be named **server.crt** and **server.key**.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md:251]:
```suggestion
      expects the certificate files to be named **server.crt** and **server.key**.
```

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md (hunk 2 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/vmware-vsphere-airgap-instructions.md:252]:
```suggestion
      expects the certificate files to be named **server.crt** and **server.key**.
```

**Comment by JPedro2** [comment]
@karl-cardenas-coding the below is actually not needed anymore. It was needed in 4.2.13 but in 4.3.6 the airgap.sh script does it already by default as it copies the certs from /opt/spectro/ssl 

Screenshot 2024-06-21 at 18 29 58

**Pull request merged**